res/values/strings.xml

```diff
@@ -335,7 +335,7 @@
     <string name="login_smtp_security">SMTP security</string>
     <string name="login_auth_method">Authorization method</string>
     <string name="login_info_oauth2_title">Continue with simplified setup?</string>
-    <string name="login_info_oauth2_text">The entered e-mail-address supports a simplified setup (OAuth2).\n\nIn the next step, please allow Delta Chat to act as your Chat with E-Mail app.\n\nThere are no Delta Chat servers, your data stays on your device!</string>
+    <string name="login_info_oauth2_text">The entered e-mail-address supports a simplified setup (OAuth 2.0).\n\nIn the next step, please allow Delta Chat to act as your chat with e-mail app.\n\nThere are no Delta Chat servers, your data stays on your device.</string>
```

Comment:
Is "Chat-With-E-Mail" a term for something?

Reply:
yip, i think it is better to put it in quotes.

Reply:
we also use the term Chat over E-Mail, for example here: https://delta.chat/de/#mit-chat-over-e-mail-zur%C3%BCck-in-die-zukunft

Reply:
Who other than internal devs pick up on that though, and how does it translate in other languages? Chat with/over/via e-mail (where via is preferable) is no more a term than communication per e-mail is, IMO.